src/main/java/com/jcabi/w3c/DefaultCssValidator.java

```diff
@@ -72,10 +77,14 @@ public ValidationResponse validate(final String css)
             ".*^/\\* JIGSAW IGNORE: [^\\n]+\\*/$.*",
             Pattern.MULTILINE | Pattern.DOTALL
         );
-        if (pattern.matcher(css).matches()) {
-            response = this.success("");
-        } else {
-            response = this.processed(css);
+        try {
```

Comment:
@dmzaytsev This new code is not mine, I just introduced the try/catch. I'll put a puzzle to someone solve it later, ok?